packages/turbo-repository/rust/src/lib.rs

```diff
@@ -35,18 +42,47 @@ pub struct Workspace {
     /// The package manager used by the workspace.
     #[napi(readonly)]
     pub package_manager: PackageManager,
+    /// The package graph for the workspace.
+    graph: PackageGraph,
 }
 
+#[napi]
 impl Package {
-    fn new(workspace_path: &AbsoluteSystemPath, package_path: &AbsoluteSystemPath) -> Self {
+    fn new(
+        name: String,
+        workspace_path: &AbsoluteSystemPath,
+        package_path: &AbsoluteSystemPath,
+    ) -> Self {
         let relative_path = workspace_path
             .anchor(package_path)
             .expect("Package path is within the workspace");
         Self {
+            name,
             absolute_path: package_path.to_string(),
             relative_path: relative_path.to_string(),
         }
     }
+
+    fn dependents(
+        &self,
+        graph: &PackageGraph,
+        workspace_path: &AbsoluteSystemPath,
+    ) -> Vec<Package> {
+        let node = WorkspaceNode::Workspace(WorkspaceName::Other(self.name.clone()));
+        let ancestors = graph.immediate_ancestors(&node).unwrap();
+
+        ancestors
+            .iter()
+            .filter_map(|node| {
+                let info = graph.workspace_info(node.as_workspace()).unwrap();
```

Comment:
I don't expect this will happen unless we have a pretty major bug in the package graph
```suggestion
                let info = graph.workspace_info(node.as_workspace())?;
```